quantum/process_keycode/process_ucis.c

```diff
@@ -118,7 +122,6 @@ bool process_ucis (uint16_t keycode, keyrecord_t *record) {
       return false;
     }
   }
-
```

Comment:
Any particular reason why this blank line was removed? If not, I'd suggest readding it as it makes this block slightly more readable.

Reply:
Nope, just a brainfart by me when committing, I attempted other ways to work around the original issue I was trying to fix that prompted this solution and didn't discard other changes properly, so this slipped by.